packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,19 +5,52 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [21.0.0]
 ### Added
-- Add `codefiTokenPricesServiceV2` ([#3600](https://github.com/MetaMask/core/pull/3600))
-  - This object can be used for the new `tokenPricesService` argument for TokenRatesController. It uses an internal API to fetch prices for tokens instead of CoinGecko.
+- Add `CodefiTokenPricesServiceV2` ([#3600](https://github.com/MetaMask/core/pull/3600), [#3655](https://github.com/MetaMask/core/pull/3655), [#3655](https://github.com/MetaMask/core/pull/3655))
+  - This class can be used for the new `tokenPricesService` argument for TokenRatesController. It uses a MetaMask API to fetch prices for tokens instead of CoinGecko.
+  - The `CodefiTokenPricesServiceV2` will retry if the token price update fails
+    - We retry each request up to 3 times using a randomized exponential backoff strategy
+    - If the token price update still fails 12 times consecutively (3 update attempts, each of which has 4 calls due to retries), we stop trying for 30 minutes before we try again.
+- Add polling by `networkClientId` to `AccountTrackerController` ([#3586](https://github.com/MetaMask/core/pull/3586))
+  - A new state property, `accountByChainId` has been added for keeping track of account balances across chains
+  - `AccountTrackerController` implements `PollingController` and can now poll by `networkClientId` via the new methods `startPollingByNetworkClientId`, `stopPollingByPollingToken`, and `stopPollingByPollingToken`.
+  - `AccountTrackerController` accepts an optional `networkClientId` value on the `refresh` method
+  - `AccountTrackerController` accepts an optional `networkClientId` value as the last parameter of the `syncBalanceWithAddresses ` method
+- Support token detection on Base and zkSync ([#3584](https://github.com/MetaMask/core/pull/3584))
+- Support token detection on Arbitrum and Optimism ([#2035](https://github.com/MetaMask/core/pull/2035))
 
 ### Changed
```

Comment:
Should we mention that `cockatiel` was added as a new dependency in 3655, and `lodash` was added as a dependency in 3586?

Reply:
Good catch, done in https://github.com/MetaMask/core/pull/3667/commits/91811aaa2795b9b2677423acd323ed5c4e89f93f